examples/tk-gas-station/README.md

```diff
@@ -0,0 +1,688 @@
+# Turnkey Gas Station SDK
+
+A reusable SDK for implementing gasless transactions using EIP-7702, Turnkey wallet management, and your own paymaster. This package provides clean abstractions and utility methods to quickly integrate with Turnkey's contracts for sponsored transaction execution.
```

Comment:
this feels looks/feels like it'd be the readme for the canonical gas-station package, as opposed to this example.

typically, our examples are centered around providing context as to what the example is demonstrating, as well as guidance on how to run. for reference: https://github.com/tkhq/sdk/tree/main/examples/with-viem